### ISSUE TITLE
Descriptions Issue

### ISSUE THREAD

**Issue by bennyj** [griff/metaz#37]
Hi Guys

I'm new to this and not very computer savvy so be gentle!

Great work , I love what you have done.
Looks great and simple to use, perfect.

One small issue since installing the latest version, I can't seem to see or edit any descriptions, short or long. Could this be something that I'm doing wrong? Any suggestions would be appreciated.

Cheers, Ben

**Comment by griff** [comment]
What do you mean exactly when you say that you can't see or edit descriptions?

Doesn't the data get loaded from the file?

Does it not get written to the file?

Are the fields actually missing from the screen?

Can you not change what is in the fields (if they are not missing)?

**Comment by bennyj** [comment]
The fields are there but I cannot see text from a file or add/edit text into them.
The field will not highlight and I am unable to type anything.

**Comment by griff** [comment]
That does sound weird.

What version of OS X are you using?

And can I get you to look in the log and either add it as a comment or email it to me at brian@maven-group.org? There is a menu item to open it: Help -> View Log

**Comment by bennyj** [comment]
No problem, I am now at work but will let you have the info asap once I get home.

**Comment by griff** [comment]
I have looked over the log and it doesn't tell me much. I did notice that descriptions get written to some of the files in the log.

Is this problem with the descriptions something that has started recently? Like when you updated to a new version of MetaZ?

**Comment by bennyj** [comment]
Yes, it has only started since installing the latest version which I installed a couple of days ago.
Its strange because when info is imported, if there is a description the amount of text available is adjusted accordingly, ie. 256 letters/symbols is the max for a short description,  there will be no text in the field but only 37 (example) spaces left.

**Comment by griff** [comment]
That it is an issue that has just started happing helps a lot. Because that means it is some recent change I made that has caused this. Could I get you to download the old alpha 12 version of MetaZ and and double check that it does in fact not have the issue? I just want to be 100% certain that it is something only alpha 13 has.

What do you mean when you say imported? Do you mean like when you get data from a search result?

And am I getting this right that the field is empty with no text shown but the counter for how many letters are left changes? And what if you click the short description and type something on the keyboard does the counter change then?

**Comment by bennyj** [comment]
I've downloaded Alpha 12 but now can't seem to open any movie files,  
they stay grey and I can't highlight them, any suggestions?

Yes, when I say imported I mean data from a search.
The field stays empty with no text shown but the counter shows that  
there should be text showing. If I click on the short description the  
field will not highlight, nothing will type and the counter does not  
change.

In light of being unable to now get alpha 12 to work I think it may be  
a problem with my system rather than the program.

**Comment by griff** [comment]
The alpha 12 issue sounds like a different unrelated problem. It sounds like either an issue with loading the plugin that does the opening of files or if you can open mp4 files but not m4v files a problem with the file types that MetaZ supports.

Could you mail me the log again so I can check what is wrong with alpha 12?

**Comment by griff** [comment]
Try moving MetaZ alpha 12 to the applications folder and maybe replacing alpha 13 with alpha 12.

**Comment by bennyj** [comment]
OK, Alpha 12 up and running again, descriptions fields both OK.
It seems the problem is only with Alpha 13.

**Comment by griff** [comment]
Great that narrows it down.

I have made a special build of MetaZ I would like for you to test if it has the issue: http://maven-group.org/metaz/MetaZ-Issue37.zip

**Comment by bennyj** [comment]
Have tested on 3 movies and all perfect, many thanks for bearing with me and taking the time to resolve the issue.
Briefly, and in simple terms (!!), what was the problem?

**Comment by griff** [comment]
Well. While I may have fixed your issue I did it by reverting some changes I had made to solve a different issue. In the version you have now the text view will lose focus when you press enter.

So if I could get you to test some other builds of MetaZ that try to fix both problems I would be most appreciative. The first build I want you to try is:  http://maven-group.org/metaz/MetaZ-Issue37-2.zip

**Comment by bennyj** [comment]
Tried this build on a couple of movies and found the same issue with the descriptions fields, issue # 37 is back.

I didn't come across the issue you mention in your previous comment about the text losing focus, I tried editing text received from a search and also adding my own description and the text seemed fine in both cases.

**Comment by griff** [comment]
So I have tried a different fix: http://maven-group.org/metaz/MetaZ-Issue37-3.zip

**Comment by bennyj** [comment]
Same descriptions issue I'm afraid.

**Comment by griff** [comment]
I am fast running out of ideas but can you try this build: http://maven-group.org/metaz/MetaZ-Issue37-4.zip

**Comment by bennyj** [comment]
Tried this build on a couple of movies and it seems to work perfectly, thanks again for your assistance.
